lib/generator/mocker.ts

```diff
@@ -12,7 +14,7 @@ export default class Mocker {
       case "array":
         return this.generateArray(paramSpec, paramName, arrItem);
       default:
-        console.log(`Unknown type ${paramSpec.type}.`);
+        log.info(`unknown type ${paramSpec.type}.`);
```

Comment:
Are you going to update all other logging place to use `log.`?

Reply:
Yes, already replaced almost all, left some to help understand the tool's running progress